akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import scala.concurrent.duration._
+
+import akka.testkit.AkkaSpec
+
+object ClusterShardingSettingsSpec {
+  val queryTimeout = 10.seconds
+}
+
+class ClusterShardingSettingsSpec
+    extends AkkaSpec(s"""
+                                                     akka.actor.provider = cluster
+                                                     akka.remote.classic.netty.tcp.port = 0
+                                                     akka.remote.artery.canonical.port = 0
+                                                     akka.cluster.sharding.shard-region-query-timeout = ${ClusterShardingSettingsSpec.queryTimeout}
```

Comment:
This test doesn't use remoting so no need for these?

Reply:
It was in ClusterShardingInternalsSpec.scala because before there wasn't a settings test - vs being there for using remoting reasons. But deleting works too.